### PR TITLE
T20376/master

### PR DESCRIPTION
**Pull request by manuq**
https://phabricator.endlessm.com/T20376

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,50 +8,10 @@ This project uses the GNOME technology stack, including GTK and WebKit for UI, a
 
 Building
 --------
-We recommend using [JHbuild] to build this code, especially if you are planning to make changes to it.
-A sample JHbuild moduleset is included in the root directory of this repository.
+We recommend using [Flapjack] to build this code, especially if you are planning to make changes to it.
+The [example.flapjack.ini] configuration file included with Flapjack is precisely made to build the `eos-knowledge-lib` and other components,
```
Comment (ptomato) [inline README.md]: There's a comma at the end of this sentence — did you intend to write more after it?

File: README.md [hunk 2]
```diff
@@ -8,50 +8,10 @@ This project uses the GNOME technology stack, including GTK and WebKit for UI, a
...
-
-If not using JHbuild you first need to clone, build, and install these other Endless repositories:
-
- - [eos-sdk]
- - [eos-shard]
- - [xapian-glib]
+Copy the `example.flapjack.ini` to `~/.config/flapjack.ini` and follow the instructions for [Flapjack] to know how to build, test, and run apps.
 
 If you wish to run the tests, you will also need to clone, build, and install [jasmine-gjs].
 Other dependencies are PyYAML for Python 3, and an SCSS compiler.
```
Comment (ptomato) [inline README.md]: You can remove this paragraph about dependencies as well. The dependencies are included in the SDK runtime.

File: README.md (hunk 1 above)
Comment (manuq) [reply README.md]: Good catch.

Comment (manuq) [reply README.md]: Good catch.

**Pull request merged**